.dlc.json

```diff
@@ -16,6 +16,7 @@
   "fallbackRetryDelay": "1000s",
   "aliveStatusCodes": [
     200,
-    401
+    401,
+    403
```

Comment:
Should we consider 302?

Reply:
I guess the tool will follow the redirect automatically, this configuration section is for "error code" that should be 4xx and 5xx

Reply:
Got it. Do you know why `https://twitter.com/ASFSkyWalking` is a dead link? The browser seems to say the other way.